src/main.cpp

```diff
@@ -113,7 +98,7 @@ void read_motor_data()
 int8_t throttle_pos = 0;
 void read_throttle()
 {
-    unsigned long pulse = pulseIn(PIN7, HIGH, 100000UL);
+    unsigned long pulse = pulseIn(PIN7, HIGH, 40000UL); // 40ms timeout, pulses should come every 20ms
```

Comment:
This whole function can be removed as it does not work

Reply:
done